guides/_subsections/common/card-catalog.md

```diff
@@ -1,9 +1,9 @@
 {% capture card-catalog-include-text %}
- The <i>Dashboard Card Catalog</i> contains a list of available cards that can be added to the dashboard. Clicking on a card will open a panel containing a preview.
+ The <i>Dashboard Card Catalog</i> contains a list of available <code>cards</code> that can be added to the dashboard. Clicking on a <code>card</code> will open a panel containing a preview.
```

Comment:
in `navigate-the-dashboard.md`, dashboard is in a code block. This subsection appears right after that and dashboard isn't in a code block... which might be confusing? Also dashboard is capitalized in line 7, but nowhere else.

Reply:
Also, dashboard is italicized in `automated-analysis.md` (line 11)